packages/node_modules/@ciscospark/react-component-activity-item/src/index.js

```diff
@@ -46,9 +46,21 @@ export default function ActivityItem(props) {
 /* eslint-disable react/no-unused-prop-types */
 /* eslint-disable-reason passing props via destructuring */
 ActivityItem.propTypes = {
-  activity: PropTypes.object,
+  activity: PropTypes.shape({
+    content: PropTypes.string,
+    displayName: PropTypes.string,
+    files: PropTypes.shape({
+      items: PropTypes.array
+    })
+  }),
   avatarUrl: PropTypes.string,
-  callData: PropTypes.object,
+  callData: PropTypes.shape({
+    callInfo: PropTypes.shape({
+      duration: PropTypes.string,
+      name: PropTypes.string
+    }),
+    status: PropTypes.string
+  }),
```

Comment:
Did you mean to have the `callData` as it's own prop? I was thinking this would be inside the `activity` object instead being it's own separate prop because the activity object has all the rest of the information. What do you think?